api/v1alpha/api/openapi.yaml

```diff
@@ -61296,8 +61284,8 @@ components:
       type: object
     AWSCloudProviderContainer:
       allOf:
-      - $ref: '#/components/schemas/CloudProviderContainer'
       - $ref: '#/components/schemas/AWSCloudProviderContainer_allOf'
+      - $ref: '#/components/schemas/AWSCloudProviderContainer_allOf_1'
```

Comment:
We do have very strange allOf of the allOf object introduced.

Reply:
During debugging I deleted the function that was flattening the object - re-added it now and this should be gone

Reply:
That was fast fix for someting that sounded like long investigation.

Ignore my comment in: https://github.com/mongodb/go-client-mongodb-atlas/pull/355#issuecomment-1430160166